src/components/Text/Text.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import "./text.css";
+
+export interface TextProps {
+  /**
+   * Text variants
+   */
+  variant?:
+    | "header"
+    | "subheader"
+    | "body"
+    | "body-small"
+    | "subtext"
+    | "header-alt"
+    | "subheader-alt"
+    | "body-alt"
+    | "subtext-alt";
+  /**
+   * Text color to use (accepts hex, eg: #212121 or #FF0000)
+   */
+  color?: string;
+  /**
+   * Font style of the text eg: italics, etc
+   */
+  fontStyle?: string;
+  /**
+   * Text to be displayed
+   */
+  text?: string;
```

Comment:
I think this can be passed as children instead of explicit text field

Reply:
So... just delete this field and remove the {text} ?

Reply:
As in can change the "text" field to "children" and the {text} to {children} so that you can do 

<Text>
   Hello
</Text>